crates/ubrn_bindgen/src/bindings/gen_rust/mod.rs

```diff
@@ -182,15 +188,12 @@ impl<'a> ComponentTemplate<'a> {
         ident("u")
     }
 
-    fn prelude(&self, ci: &ComponentInterface) -> TokenStream {
+    fn prelude(&self, _ci: &ComponentInterface) -> TokenStream {
         let runtime_module_ident = self.flavor.runtime_module();
         let runtime_ident = self.runtime_ident();
-        let crate_ident = ident(ci.crate_name());
         let uniffi_ident = self.uniffi_ident();
         quote! {
             use #runtime_module_ident::{self as #runtime_ident, uniffi as #uniffi_ident, IntoJs, IntoRust};
-            #[allow(unused_imports)]
-            use #crate_ident;
```

Comment:
This was working for single crate fixtures, because there was one `ComponentInterface` and only one crate.